cmd/podman/parse/net.go

```diff
@@ -82,45 +82,37 @@ func GetAllLabels(labelFile, inputLabels []string) (map[string]string, error) {
 		}
 	}
 	for _, label := range inputLabels {
-		split := strings.SplitN(label, "=", 2)
-		if split[0] == "" {
+		key, value, _ := strings.Cut(label, "=")
+		if key == "" {
```

Comment:
Question, but not for addressing in your PR: why is this null-key check not being done everywhere?